react/features/follow-me/actionTypes.js

```diff
@@ -0,0 +1,24 @@
+// @flow
+
+/**
+ * The type of (redux) action which updates the current known status of the
+ * Follow Me feature. Is it activated or not.
+ *
+ * {
+ *     type: SET_FOLLOW_ME_ACTIVE,
+ *     enabled: boolean
+ * }
+ */
+export const SET_FOLLOW_ME_ACTIVE = 'SET_FOLLOW_ME_ACTIVE';
```

Comment:
I notice there's already a SET_FOLLOW_ME action. Is SET_FOLLOW_ME_ACTIVE for when a remote participant has follow me active? If so a rename might be good to make it clear the active part is for a remote participant. Or maybe the two can be combined into one and the participant id used to know if follow me is enabled locally.